internal/relay/config.go

```diff
@@ -35,13 +37,11 @@ func GetNeutronChain(logger *zap.Logger, cfg *config.NeutronChainConfig, chainID
 
 func GetTargetChain(logger *zap.Logger, cfg *config.TargetChainConfig, chainID string) (*relayer.Chain, error) {
 	provCfg := cosmos.CosmosProviderConfig{
-		Key:           "",
-		ChainID:       chainID,
-		RPCAddr:       cfg.RPCAddr,
-		AccountPrefix: cfg.AccountPrefix,
-		// we don't have any needs in keys for target chain
-		// but since "KeyringBackend" can't be an empty string we explicitly set it to "test" value to avoid errors
-		KeyringBackend: "test",
+		Key:            "",
```

Comment:
default values like for fields `Key`, `GasAdjustment`, `GasPrices` and `SignModeStr` can be removed since the same values are assigned to these fields on the very instantiation

Reply:
Agreed